module/chunks/chunkVerifier_test.go

```diff
@@ -269,7 +276,7 @@ func (s *ChunkVerifierTestSuite) TestServiceEventsMismatch() {
 
 // TestServiceEventsAreChecked ensures that service events are in fact checked
 func (s *ChunkVerifierTestSuite) TestServiceEventsAreChecked() {
```

Comment:
For clarity, I would suggest to rename a couple of the existing tests, to make the conceptual difference more explicit:
```suggestion
func (s *ChunkVerifierTestSuite) TestServiceEventsAreChecked_SystemChunk() {
```